README.md

```diff
@@ -1,79 +1,63 @@
-# Auth0 WordPress Plugin
+![wp-auth0](.github/header.png)
 
-> ⚠️ **Version 5.0 of our plugin is now available in Beta.** This release adds a number of features. Do not use this in production yet, as there are a considerable number of breaking changes and some functionality remaining to be implemented. The migration sync processed is not fully functional in this release. As we move toward General Availability, please be aware that releases may contain breaking changes.
+WordPress Plugin for [Auth0](https://auth0.com) Authentication
 
-![Auth0 WordPress Plugin](https://ps.w.org/auth0/assets/banner-772x250.png)
+[![License](https://img.shields.io/packagist/l/auth0/auth0-php)](https://doge.mit-license.org/)
 
-This plugin replaces the standard WordPress login process with one powered by Auth0 that enables social, passwordless, and enterprise connection login as well as additional security, multifactor auth, and user statistics.
+:books: [Documentation](#documentation) - :rocket: [Getting Started](#getting-started) - :speech_balloon: [Feedback](#feedback)
 
-Please see the [WP.org plugin page](https://wordpress.org/plugins/auth0/) for more details on functionality.
+## Documentation
 
-## Table of Contents
+- [Download from WordPress.org](https://wordpress.org/plugins/auth0/)
+- [Documentation](https://auth0.com/docs/cms/wordpress)
 
-- [Documentation](#documentation)
-- [Installation](#installation)
-- [Getting Started](#getting-started)
-- [Contribution](#contribution)
-- [Support + Feedback](#support--feedback)
-- [Vulnerability Reporting](#vulnerability-reporting)
-- [What is Auth0](#what-is-auth0)
-- [License](#license)
+## Getting Started
 
-## Documentation
+### Requirements
 
-- [Installation](https://auth0.com/docs/cms/wordpress/installation)
-- [Configuration](https://auth0.com/docs/cms/wordpress/configuration)
-- [Troubleshooting](https://auth0.com/docs/cms/wordpress/troubleshoot)
-- [Extending](https://auth0.com/docs/cms/wordpress/extending)
+- PHP 8.0+
+- Latest version of WordPress (6+)
 
 ## Installation
 
-Please see the [installation docs](https://auth0.com/docs/cms/wordpress/installation) for detailed instructions on how to get started with Login by Auth0.
+1. Log in to an existing WordPress site as an administrator.
+2. Go to Plugins → Add New in the admin menu on the left.
+3. Search for "Login by Auth0"
+4. Click Install Now, and then Activate.
+
+Addition information can be found in the [installation docs](https://auth0.com/docs/cms/wordpress/installation).
 
 ## Getting Started
```

Comment:
Should this be renamed to `Configure the SDK` and be a h3 so it's folded under `Getting Started`?
```suggestion
### Configure the SDK
```